src/core/manager.c

```diff
@@ -3083,7 +3094,7 @@ int manager_serialize(
         assert(f);
         assert(fds);
 
-        m->n_reloading++;
+        _cleanup_(manager_reloading_stopp) Manager *reloading = manager_reloading_start(m);
```

Comment:
hmm, still not a fan of mixing variable declarations with function calls in the same line